packages/terra-clinical-header/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Breaking Changes
+  * Changed `title` prop to `text` prop.
+  * Added `level` prop to allow users to render different heading levels.
+  * Added `id` prop. 
```

Comment:
Can these be updated to be more descriptive like how they are in the upgrade guide below? When consumers look at these, they know what was changed but hard to understand why the change is made and how it would impact them.

Reply:
Updated [53ced76](https://github.com/cerner/terra-clinical/pull/792/commits/53ced7677e4fd6eb2b3bea9465b779d456355f14)